engine/src/main/java/org/hibernate/search/bridge/KeepSafeFromAnalyzerBridge.java

```diff
@@ -0,0 +1,17 @@
+/*
+ * Hibernate Search, full-text search for your domain model
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+package org.hibernate.search.bridge;
+
+
+/**
+ * This interface is used to mark the bridges we want to keep safe from the analyzers.
+ *
+ * @author Guillaume Smet
+ */
+public interface KeepSafeFromAnalyzerBridge {
```

Comment:
I didn't realize that the other bridges are in a public package, `org.hibernate.search.query.dsl` is still public so it should do, but let's see what @Sanne  thinks